backtrace-library/src/main/java/backtraceio/library/models/json/BacktraceReport.java

```diff
@@ -167,40 +162,53 @@ public BacktraceReport(
      * Create new instance of Backtrace report to send a report
      * with application exception, attributes and attachments
      *
-     * @param exception       current exception
+     * @param throwable       current throwable
      * @param attributes      additional information about application state
      * @param attachmentPaths path to all report attachments
      */
-    public BacktraceReport(
-            Exception exception,
-            Map<String, Object> attributes,
-            List<String> attachmentPaths) {
-
+    public BacktraceReport(Throwable throwable, Map<String, Object> attributes, List<String> attachmentPaths) {
         this.attributes = CollectionUtils.copyMap(attributes);
         this.attachmentPaths = CollectionUtils.copyList(attachmentPaths);
-        this.exception = this.prepareException(exception);
+        this.originalException = throwable;
+        this.exception = this.prepareException(throwable);
```

Comment:
I think this 2 should be combined
```
if (throwable != null) {
    this.exception = this.prepareException(throwable);
    this.classifier = getExceptionClassifier(throwable);
}
```

Reply:
Adjusted the code to better explain what we do now